sources/software/index.rst

```diff
@@ -27,7 +27,12 @@ The NBD's main technologies are:
 
 - modtox AI ( `modtox AI docs  <https://nostrumbiodiscovery.github.io/modtox/>`_)
 
-- schordingers platform ( `schrodinger docs  <https://www.schrodinger.com/>`_) 
+- schrodinger platform ( `schrodinger docs  <https://www.schrodinger.com/>`_)
+
+- GROMACS ( `GROMACS docs <https://manual.gromacs.org/documentation/>`_)
+
+- PLUMED ( `PLUMED docs <https://www.plumed.org/doc>`_)
```

Comment:
The whole summary is full of inconsistent spaces around parenthesis. It's purely cosmetic, but if someone has time to fix it, it would work wonders for my OCD. :D